zeppelin-plugins/launcher/k8s-standard/src/main/java/org/apache/zeppelin/interpreter/launcher/Kubectl.java

```diff
@@ -0,0 +1,157 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.zeppelin.interpreter.launcher;
+
+import com.google.common.annotations.VisibleForTesting;
+import com.google.gson.Gson;
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import org.apache.commons.exec.*;
+import org.apache.commons.io.IOUtils;
+
+import java.io.*;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class Kubectl {
+  private final Logger LOGGER = LoggerFactory.getLogger(Kubectl.class);
+  private final String kubectlCmd;
+  private final Gson gson = new Gson();
+  private String namespace;
+
+  public Kubectl(String kubectlCmd) {
+    this.kubectlCmd = kubectlCmd;
+  }
+
+  /**
+   * Override namespace. Otherwise use namespace provided in schema
+   * @param namespace
+   */
+  public void setNamespace(String namespace) {
+    this.namespace = namespace;
+  }
+
+  public String getNamespace() {
+    return namespace;
+  }
+
+  public String apply(String spec) throws IOException {
+    return execAndGet(new String[]{"apply", "-f", "-"}, spec);
```

Comment:
`kubectl apply -f` does not work if there is a `generateName` in the interpreter spec yaml. Its better to use `kubectl create -f` if resources are being created. Please find the discussion [here](https://github.com/kubernetes/kubernetes/issues/44501#issuecomment-294255660).

Reply:
I didn't know about `generateName`. Thanks for pointing this out with discussion link. 
I agree using `create -f`. While this PR works itself, I'd like to merge this first and then follow up `apply -> create` in a separate PR. Does it sounds like a plan?

Reply:
Okay... Thanks

Reply:
Created https://issues.apache.org/jira/browse/ZEPPELIN-3954. I'll follow up this issue.